### PR TITLE
rearrange ATO pages

### PR DESCRIPTION
**Pull request by afeld**
This pull request splits up and rearranges the ATO-related pages.

screen shot 2016-01-05 at 12 10 26 pm

I only made very minor changes to the content itself, so I would review this pull request for the _organization_ of the content, rather than whether each page is "complete" or not.

### CODE REVIEW

File: _pages/ato/levels.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Levels of ATO
+---
+
+ATOs are broken down to the following levels:
+
+* Open Data
+* FISMA Low
+* FISMA Medium
+* FISMA High
```
Comment (afeld) [inline _pages/ato/levels.md]: /cc https://github.com/18F/before-you-ship/issues/44

File: _pages/ato.md [hunk 2]
```diff
@@ -1,19 +1,8 @@
...
-        * Good news is that agency head can delegate that authority at their choosing.
-    * This agency-specific process then binds agency heads with the process that they must then follow.
-        * In practice, what this means is setting up a series of controls for each new system that will be launched.
-* 18F simplifies this process by implementing the bulk of the controls at the infrastructure level to the AWS account. This is reflected in a baseline minimum controls we’ve created.
-    * Controls can range from human controls to business processes to mechanical ones.
-* Before a system is made publicly accessible on the Internet, it must go through either the full ATO process or a 90-Day Authority to Test. If either are not yet complete, the system must have some level of authentication required for a user to enter.
+An Authority to Operate (ATO) is a complicated security review process that is required for deploying any web service on the public web.
```
Comment (afeld) [inline _pages/ato.md]: Definitely going to be iterating on this description.

File: _config.yml [hunk 3]
```diff
@@ -42,9 +42,22 @@ navigation:
...
+  - text: Levels
+    url: levels/
+    internal: true
+  - text: Checklist
+    url: checklist/
+    internal: true
+  - text: Authority to Test
+    url: authority-to-test/
+    internal: true
 - text: Infrastructure
```
Comment (DavidEBest) [inline _config.yml]: There isn't a link for the tips page. Should there be, or is just a holding place for that text at the moment?

File: _pages/ato.md [hunk 4]
```diff
@@ -38,18 +27,4 @@ While you’re in the ATO process, the following things will occur. These are la
...
-* What does it do?
-* How does it move information around?
-* What does it accomplish by doing it?
-
-## Optional - 90-day Authority to Test
-
-* 18F offers a 90 day authority to test security clearance process.
-    * Estimated to take approximately 2 weeks to get materials together and information security team has a service level agreement of 2 weeks to approve a completed system security plan, do the testing, and give a 90 day authorization.
-    * The 90 day ATO can be re-upped as long as the security boundary hasn’t changed; the automatic scans must not show any new vulnerabilities.
-* [Sensitive PII](../security/pii) is documented here; this process is not analogous to the PIA/SORN process and they must be handled separately
```
Comment (DavidEBest) [inline _pages/ato.md]: Is this PII reference here part of the Authority To Test section? It didn't make it over into the new page, but I can't tell if it should really be part of this section.

**Comment by leahbannon** [comment]
Maybe it would make more sense to pull my page into the PM guide? I'm having trouble tracking down how stuff was split up/what stayed and what was cut, but I think I want to keep that high level overview intact for PMs.

**Comment by DavidEBest** [comment]
@leahbannon It makes sense to give a brief overview of the ATO process in the PM guide, but I'd avoid going into too much detail. There has been an effort recently to consolidate ATO documentation to this repo, we should be slow to create new, duplicative documentation.

File: _pages/ato.md (hunk 4 above)
Comment (afeld) [inline _pages/ato.md]: After some investigation, seems like it was a mistake that it was in that section in the first place. Good catch, though!

File: _config.yml (hunk 3 above)
Comment (afeld) [inline _config.yml]: Whoops! I've been maintaining this nav list manually (for various reasons), so this was bound to happen.

**Comment by NoahKunin** [comment]
Doesn't really make sense to me this way. Not sure we should focus on InfoArch until we have a new process to actually document.

**Comment by afeld** [comment]
@NoahKunin Can you elaborate? My feeling is that we are still far from having documentation that is complete, but at least breaking it into buckets will help with organizing the information we _do_ have. Maybe these just aren't the _right_ buckets...?

**Comment by leahbannon** [comment]
Based on conversations with @afeld, I'm 👍 on merging this and then iterating if it doesn't work.

**Comment by DavidEBest** [comment]
Might be a bit premature (given the size of some of the resulting pages), but I'm :+1: on merging & iterating. Is there a particular part that doesn't make sense, @NoahKunin?

**Comment by afeld** [comment]
FWIW, since we don't have tables of contents to see at a glance what's included in a page, I've been favoring smaller pages that direct you between them.

**Comment by dlapiduz** [comment]
:+1: on my end too... I agree with @leahbannon that we can iterate later and with @afeld that more smaller pages > fewer longer ones

File: _config.yml (hunk 3 above)
Comment (afeld) [inline _config.yml]: Fixed!

**Comment by afeld** [comment]
@NoahKunin I know you're busy with a lot of other things, but since this is blocking some other work and others seem to be in support of it, I'm going to merge. More than happy to address any feedback in a follow-up pull request.

**Pull request merged**